### PR TITLE
Fix: update comparison page to use PNPM's up-to-date repo

### LINKED ISSUE #372: Update npm vs yarn comparison
Hello,

I'd like to try to update the https://yarnpkg.com/compare/ page. What repository was used so that I can do an up to date comparison?

Thomas

### PR DESCRIPTION
**Pull request by BYK**
**Summary**

Fixes #372 by redirecting the `/compare` page to PNPM's benchmark
repo. Until we have a contionously running benchmark suite for all
package managers, this is a good enough solution that provides a lot
more recent stats.

### CODE REVIEW

**Comment by Haroenv** [comment]
Deploy preview failed.

Built with commit 8a6315428d7b1cc92ca8397f6c6eb2186ade8bbf

https://app.netlify.com/sites/yarnpkg/deploys/59b145dc7960b1516d39c27b

File: _redirects [hunk 1]
```diff
@@ -21,6 +21,10 @@ layout: null
 /latest-rc.msi        https://github.com/yarnpkg/yarn/releases/download/v{{site.latest_rc_version}}/yarn-{{site.latest_rc_version}}.msi          302
 /latest-rc.deb        https://github.com/yarnpkg/yarn/releases/download/v{{site.latest_rc_version}}/yarn_{{site.latest_rc_version}}_all.deb      302
 /latest-rc.rpm        https://github.com/yarnpkg/yarn/releases/download/v{{site.latest_rc_version}}/yarn-{{site.latest_rc_version}}-1.noarch.rpm 302
+
+# Comparison page to PNPM's benchmark repo
+/compare https://github.com/pnpm/node-package-manager-benchmark 302
```
Comment (Haroenv) [inline _redirects]: also redirect `:lang/compare` and `lang/:lang/compare`

**Comment by BYK** [comment]
@Haroenv good to go now?

**Comment by Haroenv** [comment]
Almost, the css fails now because you didn't remove the place that file is imported: 

3:33:59 PM: Conversion error: Jekyll::Converters::Scss encountered an error while converting 'css/main.scss':
File to import not found or unreadable: _compare.scss. Load path: /opt/build/repo/_sass on line 39

**Comment by Haroenv** [comment]
After that, LGTM

**Comment by Haroenv** [comment]
Just waiting for the deploy preview to work now 😄

**Comment by BYK** [comment]
Woohoo

**Pull request merged**